src/initiation_protocols/login_init.rs

```diff
@@ -58,18 +58,12 @@ fn confirm_login(
 
     //protocol
     login_success(conn_id, messenger.clone(), player.clone());
-    join_game(conn_id, messenger.clone());
-    set_position(conn_id, messenger.clone(), player.clone());
 
-    messenger
-        .send(MessengerOperations::Subscribe(SubscribeMessage { conn_id }))
-        .unwrap();
-
-    //report current state to player (soon to be in it's own component for reuse)
-    //the only state we keep right now is players
+    //update the gamestate with this new player
```

Comment:
Lot of this logic got moved into the player service since it needs to player data to operate. We should rethink this file and the login process in general in 2020